pytition/petition/templates/petition/transfer_petition.html

```diff
@@ -0,0 +1,117 @@
+{% extends "layouts/layout.html" %}
```

Comment:
When I tested, it felt strange to not have the same navbar than on my dahsboard (the black one).
I don't know what you think and whether this is easy to change

Reply:
Is it better like this?

Reply:
Perfect! :)